backend/services/notificationService.test.ts

```diff
@@ -616,7 +619,7 @@ describe('Notification service', () => {
       await notificationService.sendBookingCreationEmails(context, 'A_USER', createEmail)
 
       expect(notifyApi.sendEmail).toHaveBeenCalledWith(
-        config.notifications.bookingCreationConfirmationPrison,
+        config.notifications.bookingCreationConfirmationPrisonWithCourtEmailAddress,
```

Comment:
will need to add a test to cover the other situation

Reply:
added